scripts/kubernetes/logo_horizon.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+## Push custom code into the Horizon container
+## TODO - add bootstrap abilities to Horizon Chard
+
+## assumes this runs at a location that can access to kubectl
+
+export HORIZON=$(kubectl get pod -o name -n openstack | sed 's/pods\///' | grep horizon)
```

Comment:
why not get this via the labels? - this method could hit a db-sync/init real easily.